src/protocols/did_exchange/helper.rs

```diff
@@ -130,6 +130,10 @@ pub fn get_did_exchange_message(
     let exchange_request: MessageWithBody<DidDocumentBodyAttachment<Base64Container>> =
         MessageWithBody {
             body: Some(DidDocumentBodyAttachment {
+                comment: match message.base_message.body.get("comment") {
+                    Some(label) => label.as_str().map(|v| v.to_string()),
+                    None => None,
+                },
```

Comment:
We could also use the Rust version of `flatmap` here. ^^

```suggestion
                comment: message
                    .base_message
                    .body
                    .get("comment")
                    .and_then(|label| label.as_str().map(|v| v.to_string())),
```

Basically works like a usual flatmap, even if the name is a bit obscure. It kind of makes sense if you look at the other functions in that context (`and`, `or`, `or_else`) but that naming is a bit... >.>